src/probeinterface/utils.py

```diff
@@ -139,7 +139,24 @@ def generate_unique_ids(min: int, max: int, n: int, trials: int = 20) -> np.arra
     return ids
 
 
-def get_auto_lims(probe, margin=40):
+def get_auto_lims(probe: Probe, margin: float = 40.0) -> tuple[float, float, float]:
+    """
+    Compute the boundaries of a given probe, considering its contour and an optional margin.
+    The function is designed to handle both planar and tridimensional probes.
```

Comment:
```suggestion
    The function is designed to handle both planar and three-dimensional probes.
```
this is really minor: tridimensional is a word, but it isn't really used (at least in US English). The more common would be 3D or three-dimensional. Just to make this docstring as user friendly as possible I would just use the more common English rather than the more scientific word.

Reply:
Otherwise this looks good to me!